src/qt/bitcoingui.cpp

```diff
@@ -341,6 +341,9 @@ void BitcoinGUI::createActions()
     m_close_wallet_action = new QAction(tr("Close Wallet..."), this);
     m_close_wallet_action->setStatusTip(tr("Close wallet"));
 
+    m_close_all_wallets_action = new QAction(tr("Close All Wallets..."), this);
```

Comment:
I'm not sure if `...` is appropriate here, just for a confirmation dialog?

Reply:
I know the feeling, I'm just following the "rule".